judge/views/contests.py

```diff
@@ -711,6 +713,46 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class ContestOfficialRanking(ContestRankingBase):
+    template_name = 'contest/official-ranking.html'
+    tab = 'official_ranking'
+
+    def get_title(self):
+        return _('%s Official Rankings') % self.object.name
+
+    def get_ranking_list(self):
+        def display_user_problem(points, contest_problem):
+            return format_html(
+                u'<td class="{state}">{points}</td>',
+                state=BaseContestFormat.best_solution_state(points, contest_problem.points),
```

Comment:
Because the `contest_problem.points` is not always the same as the points in the official contest, I think this should be changed to something like `max_points` of the current problem.